tests/update_test.go

```diff
@@ -882,3 +882,52 @@ func TestSaveWithHooks(t *testing.T) {
 		t.Errorf(`token content should be "token2_encrypted", but got: "%s"`, o2.Token.Content)
 	}
 }
+
+// only postgres, sqlserver, sqlite support update from
+func TestUpdateFrom(t *testing.T) {
```

Comment:
the test failed when running all test cases

Reply:
@jinzhu It seems that the problem is not these tests, but the new versions of the drivers that were taged and are used to run the tests in the ci (in master the tests passed because the ci was run before the new tags were created). For the postgree driver the v1.5.3 tag was created but the tests do not pass using this version (they do not pass because of the changes in the commit https://github.com/go-gorm/postgres/commit/7ba909e839c99062711d10be7f1a03f19f8d9694 specifically). In the same way, in the mysql driver the v1.5.2 tag was created but the tests don't pass since the changes of the commit https://github.com/go-gorm/mysql/commit/2a61ba01095c98f4610c8d841050ec39c04b7ea2. In my fork (https://github.com/ditrit/gorm) you can see the master tests do not pass with the new driver tags

Reply:
@FrancoLiberali The test failure has nothing to do with this PR. You seem to have investigated the cause of the failure. Are you interested in creating other PRs to fix it?

Reply:
@a631807682 To be honest, I only ran the tests on the different driver's commits to find out from which one the tests don't work, but I don't know what is the purpose of the changes in those commits to fix the generated problems